### ISSUE TITLE
Test timeout error

### ISSUE THREAD

**Issue by vforv** [hemerajs/hemera-testsuite#6]
I cannot figure it out why test not working... Here is reproduced error:
https://github.com/vforv/hemera-error

**Comment by StarpTech** [comment]
Hi @vforv I checked your code and I found out that you never called the callback from server method { topic: 'user', cmd: 'login' } search for `cb` and you can find only once.

**Comment by StarpTech** [comment]
Remove in both stubs the objects and it works. actStub expect a schema.

```js
            // stub act calls
            actStub.stub({
                topic: 'mongo-store',
                cmd: 'find',
                collection: 'users',
                //query: { email: 'vladimir@some.com' }
            }, null, { result: [] })

            actStub.stub({
                topic: 'mongo-store',
                cmd: 'create',
                collection: 'users',
                //data: userData
            }, null, "Mail sent")
```

**Comment by StarpTech** [comment]
@vforv I will think about the current behavior. It should be possible to mock the exact pattern.

**Comment by StarpTech** [comment]
Hi @vforv I revert my old changes and released hemera-testsuite@1.4.0. I tested with your repo and everything works great.

**Comment by vforv** [comment]
Great, thank you. It works now but I have another problem. I will close this issue and open a new one.
